### DISCUSSION TITLE
Different codes on mobile and desktop

### DISCUSSION THREAD

**Discussion by petaqui** [ente-io/ente#4168] [Q&A] [answered]
Good morning
Something really weird happens to me. The windows app shows one code, and android shows a different one. The ones on windows are wrong. Of course I'm using the same account on both... really weird.

**Comment by nvgoldendog** [comment]
Check your clock on both the Windows Machine and Android. Codes are generated based on precise time.

**Comment by petaqui** [comment, marked as answer]
Weird, but after updating the app to latest version isn't happening anymore
